src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
```

Comment:
- [ ] Awesome job for creating an initialState. Now let's kindly make sure the initialState has a state value `books` that contain an empty array as requested in your requirements.
![image](https://github.com/mosidrum/bookstore/assets/63087491/45d359ea-bea7-4e80-8fc7-1e47c7aa1a17)

**Hint:**
```
initialState: {
    books: [],
  },
```